lib/pos_blockchain.go

```diff
@@ -1405,7 +1779,68 @@ func (bc *Blockchain) getMaxSequentialBlockHeightAfter(startingHeight uint64) ui
 	return maxSequentialHeightWithBlocks
 }
 
+func (bc *Blockchain) GetProofOfStakeGenesisQuorumCertificate() (*QuorumCertificate, error) {
+	finalPoWBlock, err := bc.GetFinalCommittedPoWBlock()
+	if err != nil {
+		return nil, err
+	}
+
+	aggregatedSignature, signersList, err := BuildQuorumCertificateAsProofOfStakeCutoverValidator(finalPoWBlock.Header.Height, finalPoWBlock.Hash)
+	if err != nil {
+		return nil, err
+	}
+
+	qc := &QuorumCertificate{
+		BlockHash:      finalPoWBlock.Hash,
+		ProposedInView: finalPoWBlock.Header.Height,
```

Comment:
I see we set ProposedInView here, which is good. But again I'm not seeing how ew get past the error in getLineageFromCommittedTip().

Reply:
Yeah, this line is safe. I updated it to set `ProposedInView: finalPoWBlock.Header.GetView()` so it's clearer too